Makefile

```diff
@@ -367,8 +367,25 @@ upload-db:
 TARGET ?= staging
 
 
-release-list release deploy:
-	@echo $@ not yet implemented
+release-list:
+	@git ls-remote --tags | grep -o 'refs/tags/v[0-9]*\.[0-9]*\.[0-9]*' | sort -r | grep -o '[^\/]*$$'
+
+
+install-releasedeps:
+	@if [ -d "$DIRECTORY" ]; then \
```

Comment:
$DIRECTORY is not declared anywhere.

Reply:
good catch - no longer relevant since ive removed the target altogether